dask/groupby-dask2.py

```diff
@@ -38,8 +36,12 @@
   exit(0) # not yet implemented #171, currently groupby's dropna=False argument is ignored
 
 print("using disk memory-mapped data storage" if on_disk else "using in-memory data storage", flush=True)
-#x = dd.read_parquet(src_grp, engine="fastparquet") if on_disk else
-x = dd.read_csv(src_grp, dtype={"id1":"category","id2":"category","id3":"category","id4":"Int32","id5":"Int32","id6":"Int32","v1":"Int32","v2":"Int32","v3":"float64"})
+#x = dd.read_parquet(src_grp, engine="pyarrow") if on_disk else
+x = dd.read_csv(
+  src_grp,
+  dtype={"id1":"category","id2":"category","id3":"category","id4":"Int32","id5":"Int32","id6":"Int32","v1":"Int32","v2":"Int32","v3":"float64"},
+  engine="pyarrow"
```

Comment:
I also changed this to use pyarrow. It's just much faster and installed with `dask[complete]` already.

Reply:
this speedup does not count towards the benchmarks since we're persisting before the counter starts. This makes it a little more bearable to iterate on this, though